api/src/Controller/Group/ResetInviteKey.php

```diff
@@ -43,8 +43,9 @@ public function index(
         string $id,
         #[CurrentUser] User $currentUser
     ): Response {
-        $this->denyAccessUnlessGranted('ROLE_USER');
```

Comment:
We still want to deny access to this endpoint if the user is not logged in. I think that line 46 should be kept

Reply:
Ah I didn't intend to remove this, good catch. Have restored it.